### PR TITLE
Add policy for disabling failure accrual

### PR DESCRIPTION
**Pull request by klingerf**
In #859 I added the ability to configure router clients' failure accrual policies, but I didn't include the ability to disable failure accrual altogether. This branch adds a new type of failure accrual -- `none` -- which can be used to disable failure accrual.

### CODE REVIEW

**Comment by klingerf** [comment]
@olix0r Good suggestions -- I like `none` so I went with that, but happy to adjust. Also let me know if any of the class/object names in NoneInitializer.scala should change.

File: linkerd/docs/failure_accrual.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ it traffic in increasing intervals between 5 seconds and 5 minutes.
 
 Key | Default Value | Description
 --- | ------------- | -----------
-kind | _required_ | Either [`io.l5d.consecutiveFailures`](#consecutive-failures), [`io.l5d.successRate`](#success-rate) or [`io.l5d.successRateWindowed`](#success-rate-windowed).
+kind | _required_ | Either [`io.l5d.consecutiveFailures`](#consecutive-failures), [`io.l5d.successRate`](#success-rate), [`io.l5d.successRateWindowed`](#success-rate-windowed) or [`none`](#none).
```
Comment (adleong) [inline linkerd/docs/failure_accrual.md]: oxford comma

File: linkerd/failure-accrual/src/main/scala/io/buoyant/linkerd/failureAccrual/NoneInitializer.scala [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import io.buoyant.linkerd.{FailureAccrualConfig, FailureAccrualInitializer}
+
+class NoneInitializer extends FailureAccrualInitializer {
+  val configClass = classOf[NullConfig]
+  override def configId = "none"
+}
+
+object NoneInitializer extends NoneInitializer
+
+case class NullConfig() extends FailureAccrualConfig {
```
Comment (adleong) [inline linkerd/failure-accrual/src/main/scala/io/buoyant/linkerd/failureAccrual/NoneInitializer.scala]: Don't need to make this a case class, and you can call it NoneConfig for consistency.

File: linkerd/failure-accrual/src/test/scala/io/buoyant/linkerd/failureAccrual/noneTest.scala [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package io.buoyant.linkerd.failureAccrual
+
+import com.twitter.finagle.service.exp.FailureAccrualPolicy
+import com.twitter.finagle.util.LoadService
+import io.buoyant.linkerd.FailureAccrualInitializer
+import io.buoyant.test.FunSuite
+
+class NoneTest extends FunSuite {
```
Comment (adleong) [inline linkerd/failure-accrual/src/test/scala/io/buoyant/linkerd/failureAccrual/noneTest.scala]: Capitalize file name to match class name.

**Pull request merged**